app/src/androidTest/java/com/github/houseorganizer/houseorganizer/CalendarActivityTest.java

```diff
@@ -151,4 +152,20 @@ public void monthlyViewOpensRecyclerView() throws InterruptedException {
         }
         onView(withText("title")).inRoot(isDialog()).check(matches(isDisplayed()));
     }
+
+    @Test
+    public void monthNavigationWorks() throws InterruptedException {
+        String yearMonthText = YearMonth.now().plusMonths(1).format(DateTimeFormatter.ofPattern("LLLL uuuu"));
+        String yearMonthText2 = YearMonth.now().format(DateTimeFormatter.ofPattern("LLLL uuuu"));
+        // The wait is here in order to wait for the refreshing of the calendar
+        onView(withId(R.id.calendar_screen_view_change)).perform(click());
+        Thread.sleep(1000);
```

Comment:
Summoning the god of thread.sleeps @sansive

Reply:
Thanks! I'll change it on my PR